src/main/scala/ophan/google/indexing/observatory/model/Site.scala

```diff
@@ -22,7 +22,7 @@ object Sites {
   )
 
   val BBC = Site(
-    url = "https://www.bbc.co.uk",
+    url = "https://www.bbc.co.uk/",
```

Comment:
Is the trailing `/` significant in any way?

Reply:
I just realised, when looking at the logging, that the site urls were inconsistent - some had ending slashes and others didn't! So I just made them consistent.